base_export_manager/models/ir_exports_line.py

```diff
@@ -53,104 +62,106 @@ def _default_model1_id(self):
         """Default model depending on context."""
         return self.env.context.get("default_model1_id", False)
 
-    @api.multi
-    @api.depends("field1_id", "field2_id", "field3_id")
+    @api.one
```

Comment:
I don't see the point for the extra work when doing the port to v9.
And when asking to reconsider deprecation, all comments I had were against keeping it. So I don't see must space for any hopes on that.